docs/Upgrade-Guide/upgrade_to_3.12.md

```diff
@@ -42,18 +43,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
     #gluster volume status
 ```
 
-7. Self-heal all gluster volumes by running
+7. Start glustereventsd service (if it was enabled before), either using the command below, or through other means,
+```sh
+    #systemctl start glustereventsd
+```
+
+8. Self-heal all gluster volumes by running
 ```sh
     #for i in `gluster volume list`; do gluster volume heal $i; done
 ```
 
-8. Ensure that there is no heal backlog by running the below command for all volumes
+9. Ensure that there is no heal backlog by running the below command for all volumes
```

Comment:
Verify that there are no heal backlog by running the command for all the volumes

Reply:
Done